docs/api.rst

```diff
@@ -3943,6 +3943,193 @@ Refreshing asset icons
    :statuscode 500: Internal rotki error
 
 
+Get asset mappings for a location
+=================================
+
+.. http:post:: /api/(version)/assets/mappings
+
+    Doing a POST on the asset mappings endpoint will return all the assets mappings of the given location.
+
+    **Example Request**
+
+    .. http:example:: curl wget httpie python-requests
+
+        POST /api/1/assets/mappings/ HTTP/1.1
+        Host: localhost:5042
+        Content-Type: application/json;charset=UTF-8
+
+        {
+          "location": "binance",
+          "offset": 20,
+          "limit": 3
+        }
+
+    :reqjson str location: Location for which these mappings are to be saved.
```

Comment:
This is for querying. Also shouldn't it be optional and if not given provide all location mappings?
```suggestion
    :reqjson str location[optional]: If given filter the returned mappings only for the location.
```